src/hooks/useDydxClient.tsx

```diff
@@ -209,6 +213,7 @@ const useDydxClientContext = () => {
     networkConfig,
     compositeClient,
     faucetClient,
+    isConnected,
```

Comment:
is this essentially the same as checking `!!compositeClient`, if so maybe we don't need the state?